### PR TITLE
(fixes #148 #158) Fix api doc and upgrade to scala 2.10.5

### LINKED ISSUE #148: Update api doc to use the multi-project build

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -4,8 +4,12 @@ addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "0.5.1")
 
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.5.3")
+addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.5.2")
```
Comment (sethyates) [inline project/plugins.sbt]: Why the downgrade?

Comment (realstraw) [inline project/plugins.sbt]: oops, was testing something, I'll change it back.

**Comment by realstraw** [comment]
Changed the version back

File: build.sbt [hunk 2]
```diff
@@ -74,7 +65,7 @@ lazy val commonSettings = Seq(
   version := hyperionVersion,
   scalaVersion := "2.11.7",
   crossScalaVersions := Seq(
-    "2.10.4",
+    "2.10.5",
```
Comment (sethyates) [inline build.sbt]: I feel like we should pull these versions out and move up as constants next to hyperionVersion, especially given one is repeated.  So I'd add above:
```
val scala10Version = "2.10.5"
val scala11Version = "2.11.7"
```

**Comment by sethyates** [comment]
LGTM :sparkles: :sparkles: :sparkles: :+1: (only a nit remaining, which isn't a blocker)

**Comment by realstraw** [comment]
closed by 59c01f722c442f33878febc9b3a43940ce16d9b2

**Pull request closed without merging**